app/controllers/main/routes.py

```diff
@@ -45,42 +46,76 @@ def events(selectedTerm=None):
         currentTime = currentTime,
         user = g.current_user)
 
-@main_bp.route('/profile/<username>', methods = ['GET'])
-def profilePage(username):
-    if not g.current_user.isCeltsAdmin and not g.current_user.isCeltsStudentStaff and g.current_user.username != username:
-        return "Access Denied", 403
+@main_bp.route('/profile/<username>', methods=['GET'])
+def viewVolunteersProfile(username):
+    """
+    This function displays the information of a volunteer to the user
+    """
+    try:
+        User.get(User.username == username)
+    except Exception as e:
+        print(e)
```

Comment:
You don't need to print this one. There's no extra information we would get from that output

Reply:
Deleted